templates/database.html

```diff
@@ -55,7 +55,5 @@ <h2>Queries</h2>
 {% endif %}
 
 <p>Download SQLite DB: <a href="/{{ database }}-{{ database_hash }}.db">{{ database }}.db</a></p>
-
-{% include "_codemirror_foot.html" %}
-
+<p>THIS IS THE DATABASE PAGE</p>
```

Comment:
😆

Reply:
lol i'm leaving myself breadcrumbs, ok! it's out now